controllers/constant/crossplane.go

```diff
@@ -39,7 +39,7 @@ metadata:
     ibm-crossplane-provider: {{ .CrossplaneProvider }}
 spec:
   ignoreCrossplaneConstraints: false
-  package: 'quay.io/opencloudio/ibm-crossplane-bedrock-shim-config:1.0.0'
+  package: 'quay.io/opencloudio/ibm-crossplane-bedrock-shim-config:1.1.0'
```

Comment:
@PawelKopel 

Is this image will be used in some deployment from crossplane. Since it is using tag instead of digest, I am curious how it works with image mirroring in an airgap environment.

Reply:
Thanks for pointing this out, we are now working on a change to use tag provided in crossplane's CSV.
Please do not merge this PR.